README.md

```diff
@@ -171,6 +171,23 @@ To modify the background of the current slide, **jekyll-reveal.js** simplifies t
 This slide has a white background
 ```
 
+#### Image backgrounds
+
+You can also set image backgrounds:
+
+```markdown
+
+# Slide
+
+<backgroundimage>{{ site.github.url }}/images/image.jpg</backgroundimage>
+<backgroundimageopacity>0.25</backgroundimageopacity>
+
+This slide has an image background
+
+```
+
+Note: `{{ site.github.url }}` expands to the URL of your hosted site, but you could also use remote URLs.
```

Comment:
Wouldn't `{{ site.url }}` be enough as long as this key is defined in `_config.yml`?

Might be best not to assume repos are hosted on public GitHub, as we use this on a private GitHub Enterprise installation, and others might run it on BitBucket, GitLab, or elsewhere.

Reply:
Ah yes, interesting point. I’m biased on that one, but I have a policy of not using `{{ site.url }}` as it can end up causing versioning issues where your local build drags images/files from the published version. Perhaps I could provide support for either?

Reply:
You lost me here, but you're definitely more experienced with this. @mrmanc I gave you push access so you should be able to merge the pull request, if you and @infotexture agreed on something.

Thanks so much for keeping this alive, guys!

Reply:
Cool, I’ll certainly wait for feedback on this and can explore supporting both styles. I also want to check whether `{{ site.url }}` is also populated by the `jekyll-github-metadata` plugin—I don’t *think* it is sadly.

Reply:
> I have a policy of not using `{{ site.url }}` as it can end up causing versioning issues where your local build drags images/files from the published version

In recent Jekyll versions (as of 3.3), this shouldn't be an issue, as [`site.url` is set by the development server](https://jekyllrb.com/news/#3-siteurl-is-set-by-the-development-server).

Reply:
> In recent Jekyll versions (as of 3.3), this shouldn't be an issue, as [`site.url` is set by the development server](https://jekyllrb.com/news/#3-siteurl-is-set-by-the-development-server).

Awesome… I’ve learned something there! I might not have noticed this change. In any case, reading that I’ve changed it to use the `relative_url` filter which seems safe when publishing to subdirectories and compatible with other platforms. 👍

Reply:
That sounds like a good solution.

Reply:
Thanks and sorry—I missed your comment @infotexture. Sounds like we’ve resolved this then. I don’t usually merge my own PRs but since you’ve agreed I’ll go ahead :)